gateway/ocfl_repository_gateway.py

```diff
@@ -103,8 +106,8 @@ def has_object(self, id: str) -> bool:
                 return False
             raise RepositoryGatewayError() from e
 
-    def has_staged_object_changes(self, id: str):
-        args = ["rocfl", "-r", self.storage_path, "status", id]
+    def has_staged_object(self, id: str):
+        args = ["rocfl", "-r", str(self.storage_path), "status", id]
```

Comment:
We changed the name of this method while refactoring since we discovered it's still `True` even if there are no actual staged files, just the empty staged object.

Reply:
Do we need to explicitly convert self.storage_path to a string since Python Path objects can be automatically converted to strings when needed?

Reply:
My IDE had some type-related complaint that I was trying to fix here. I will experiment with it a little.